docs/guides/lint-asyncapi.md

```diff
@@ -0,0 +1,133 @@
+---
+title: Lint AsyncAPI
+---
+
+# Lint AsyncAPI with Redocly CLI
```

Comment:
I know it's a bit outside the scope of this document, but I think a concise blurb about _what_ the AsyncAPI is (or what it's used for) would go a long way here. Even just a sentence or two with outbound links would be useful.

Reply:
We do need something like this, but it doesn't belong here in a how-to guide, and I'm not adding any other AsyncAPI content at this point. If it means that only people who know what AsyncAPI is can use it now - that's probably correct. I think if we get some feedback, we'll create a whole AsyncAPI docs section with the concepts and examples and stuff (and probably split out the reference sections from this guide, they are also in the wrong place here really but they will get us started) and the do a formal announcement.

Reply:
TL;DR - really insightful and valid point, and I'm going to ignore it !!